GitUI/CommandsDialogs/SettingsDialog/Pages/AdvancedSettingsPage.cs

```diff
@@ -16,6 +16,8 @@ public AdvancedSettingsPage()
                 new { Key = "-", Value = "-" },
                 new { Key = "(none)", Value = "" },
             };
+            cboAutoNormaliseSymbol.DisplayMember = "Key";
```

Comment:
Why not nameof()?

Reply:
➡️ Looks like you get CS8082 if you try to use `nameof(autoNormalizeSymbols[0].Key)`